Jenkinsfile.release

```diff
@@ -0,0 +1,326 @@
+                            stash name: "bin-win"
+                        }
+                    }
+                }
+                stage('alpine runtime images') {
+                    agent {
+                        label 'linux'
+                    }
+                    steps {
+                        buildRuntimeImage('alpine')
+                    }
+                }
+                stage('debian runtime images') {
+                    agent {
+                        label 'linux'
+                    }
+                    steps {
+                        buildRuntimeImage('debian')
+                    }
+                }
+            }
+        }
+
+        stage('Release') {
+            when {
+                buildingTag()
+            }
+            parallel {
+                stage('Pushing images') {
+                    agent {
+                        label 'linux'
+                    }
+                    steps {
+                        pushRuntimeImage('alpine')
+                        pushRuntimeImage('debian')
+                    }
+                }
+                stage('Creating Github Release') {
+                    agent {
+                        label 'linux'
+                    }
+                    steps {
+                        checkout scm
+                        sh 'mkdir -p dist'
+                        dir("dist") {
+                            unstash "bin-darwin"
+                            unstash "bin-linux"
+                            unstash "bin-win"
+                            githubRelease("jcsirot/compose")
+                        }
+                    }
+                }
+                stage('Publishing Python packages') {
+                    agent {
+                        label 'linux'
+                    }
+                    steps {
+                        checkout scm
+                        withCredentials([[$class: "FileBinding", credentialsId: 'pypirc-docker-dsg-cibot', variable: 'PYPIRC']]) {
+                            sh './script/release/python-package'
+                        }
+                        archiveArtifacts artifacts: 'dist/*', fingerprint: true
+                    }
+                }
+                stage('Publishing binaries to Bintray') {
+                    agent {
+                        label 'linux'
+                    }
+                    steps {
+                        checkout scm
+                        dir("dist") {
+                            unstash "bin-darwin"
+                            unstash "bin-linux"
+                            unstash "bin-win"
+                        }
+                        withCredentials([usernamePassword(credentialsId: 'bintray-docker-dsg-cibot', usernameVariable: 'BINTRAY_USER', passwordVariable: 'BINTRAY_TOKEN')]) {
+                            sh './script/release/push-binaries'
+                        }
+                    }
+                }
```

Comment:
Shall we also publish to https://download.docker.com/ ?
Sounds like a cannonical place to get docker binaries

Reply:
Yes we should but I think it would deserve a separate PR. And I prefer this PR remains functionally identical to the current release pipeline